test/e2e/plugin/go.mod

```diff
@@ -29,3 +29,5 @@ require (
 	golang.org/x/sync v0.6.0 // indirect
 	oras.land/oras-go/v2 v2.5.0 // indirect
 )
+
+replace github.com/notaryproject/notation-go => github.com/Two-Hearts/notation-go v0.0.0-20240815061530-d5e876aecb1d
```

Comment:
Leave a comment as a reminder for dependency update.

Reply:
Updated.